liquidctl/driver/commander_pro.py

```diff
@@ -0,0 +1,499 @@
+"""liquidctl drivers for Corsair Commander Pro devices.
+
+Supported devices:
+
+- Corsair Commander Pro
+- Corsair Lighting Node Pro
+
+
+NOTE:
+    This device currently only has hardware control implemented but it also supports a software control mode.
+    Software control will be enabled at a future time.
+
+
+Copyright (C) 2020–2020  Marshall Asch and contributors
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+import itertools
+import logging
+import re
+
+from enum import Enum, unique
+
+from liquidctl.driver.usb import UsbHidDriver
+from liquidctl.keyval import RuntimeStorage
+from liquidctl.pmbus import compute_pec
+from liquidctl.util import clamp, fraction_of_byte, u16be_from, u16le_from, normalize_profile
+from liquidctl.error import NotSupportedByDevice
+
+
+LOGGER = logging.getLogger(__name__)
+
+_REPORT_LENGTH = 64
+_RESPONSE_LENGTH = 16
+
+
+_CMD_GET_FIRMWARE = 0x02
+_CMD_GET_BOOTLOADER = 0x06
+_CMD_GET_TEMP_CONFIG = 0x10
+_CMD_GET_TEMP = 0x11
+_CMD_GET_VOLTS = 0x12
+_CMD_GET_FAN_MODES = 0x20
+_CMD_GET_FAN_RPM = 0x21
+_CMD_SET_FAN_DUTY = 0x23
+_CMD_SET_FAN_PROFILE = 0x25
+
+_CMD_RESET_LED_CHANNEL = 0x37
+_CMD_BEGIN_LED_EFFECT = 0x34
+_CMD_SET_LED_CHANNEL_STATE = 0x38
+_CMD_LED_EFFECT = 0x35
+_CMD_LED_COMMIT = 0x33
+
+_LED_PORT_STATE_HARDWARE = 0x01
+_LED_PORT_STATE_SOFTWARE = 0x02
+_LED_SPEED_FAST = 0x00
+_LED_SPEED_MEDIUM = 0x01
+_LED_SPEED_SLOW = 0x02
+
+_LED_DIRECTION_FORWARD = 0x01
+_LED_DIRECTION_BACKWARD = 0x00
+
+_FAN_MODE_DISCONNECTED = 0x00
+_FAN_MODE_DC = 0x01
+_FAN_MODE_PWM = 0x02
+
+
+_PROFILE_LENGTH = 6
+_CRITICAL_TEMPERATURE = 60
+_MAX_FAN_RPM = 5000             # I have no idea if this is a good value or not
```

Comment:
Could this be parameterised, made disableable, or ignored when explicitly defining value over 60*C?
I've found usecase when I'd like to set curve with highest point starting at 70*C and having only 1200 out of 1800 rpm because fan is getting too noisy at max rpm. With current code executing:
`./liquidctl --match commander set fan1 speed 40 500 70 1200 100 1250 --temperature-sensor 2`
It will max out the 1800 rpm fan few degrees over 40*C

Reply:
After thinking about this a little bit we decided to add an `--unsafe=high_tempature` flag that will increase the maximum temperature to 100 degrees. In most cases the 60 degrees will be left as a maximum ambient temperature. as an alternative you could add an extra temperature point at 1 degree bellow that max if you want to reduce the fan speed.

If you have any further issues with that it would probably be better to open a new issue for this to be addressed, but for now Im going to mark this as resolved.